tv_apps/remote-control/js/app.js

```diff
@@ -0,0 +1,144 @@
+/* global KeyNavigationAdapter, Section, Config */
+'use strict';
+
+(function(exports) {
+  var MAIN_SECTION = 'main-section';
+  var DEFAULT_ELEMENT = 'qrcode';
```

Comment:
The variable name didn't have too much meaning to me. Does it mean `DEFAULT_FOCUSED_ELEMENT`?